mobile/src/components/interviews/interviews-table/components/status-cell/status-cell.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+
+import { Text } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  text: string;
+  color?: string;
+};
+
+const StatusCell: FC<Props> = ({ text, color = 'transparent' }) => {
```

Comment:
Everything looks good to me, the only thing I meant to add padding to cell, not to status label itself

Reply:
Oh.... 
sorry, I misunderstood you

Reply:
![Screenshot from 2022-08-30 00-00-44](https://user-images.githubusercontent.com/12743599/187298177-8023d735-7d80-4862-bb59-1168e973413e.png)
Should i align text by center in other cell (Id, Name, Interviewer, Date)?

Reply:
no, we don't need it